files/default/sql/ddl/updates/1.1.0.sql

```diff
@@ -93,3 +93,185 @@ CREATE TABLE `statistic_columns` (
+
+CREATE TABLE `hopsworks`.`subjects_compatibility` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `subject` varchar(255) COLLATE latin1_general_cs NOT NULL,
+  `compatibility` ENUM('BACKWARD', 'BACKWARD_TRANSITIVE', 'FORWARD', 'FORWARD_TRANSITIVE', 'FULL', 'FULL_TRANSITIVE', 'NONE') NOT NULL DEFAULT 'BACKWARD', 
+  `project_id` int(11) NOT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `subjects_compatibility__constraint_key` UNIQUE (`subject`, `project_id`),
+  CONSTRAINT `project_idx` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE `hopsworks`.`schemas` (
+  `id` int(11) NOT NULL AUTO_INCREMENT, 
+  `schema` varchar(10000) COLLATE latin1_general_cs NOT NULL,
+  `project_id` int(11) NOT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `project_idx` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+-- add project compatibility for all projects
+INSERT INTO `subjects_compatibility` (`subject`, compatibility, project_id)
+	SELECT
+		'projectcompatibility' AS `subject`,
+		'BACKWARD' AS compatibility,
+		p.id AS project_id
+	FROM
+		`project` p;
+
+-- add inferenceschema compatibility for all projects
+INSERT INTO `subjects_compatibility` (`subject`, compatibility, project_id)
+	SELECT
+		'inferenceschema' AS `subject`,
+		'NONE' AS compatibility,
+		p.id AS project_id
+	FROM
+		`project` p;
+
+-- add inference schemas to schemas table
+REPLACE INTO `schemas`(`schema`, `project_id`)
+	SELECT 
+		(SELECT 
+				s.contents AS `schema`
+			FROM
+				`schema_topics` s
+			WHERE
+				s.name = 'inferenceschema'
+					AND s.version = 1),
+		p.id AS project_id
+	FROM
+		`project` p;
+
+REPLACE INTO `schemas`(`schema`, `project_id`)
+	SELECT 
+		(SELECT 
+				s.contents AS `schema`
+			FROM
+				`schema_topics` s
+			WHERE
+				s.name = 'inferenceschema'
+					AND s.version = 2),
+		p.id AS project_id
+	FROM
+		`project` p;
+
+-- create table "subjects"
+CREATE TABLE `subjects` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `subject` VARCHAR(255) COLLATE LATIN1_GENERAL_CS NOT NULL,
+    `version` INT(11) NOT NULL,
+    `schema_id` INT(11) NOT NULL,
+    `project_id` INT(11) NOT NULL,
+    `created_on` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (`id`),
+    KEY `project_id_idx` (`project_id`),
+    KEY `created_on_idx` (`created_on`),
+    CONSTRAINT `project_idx` FOREIGN KEY (`project_id`)
+        REFERENCES `project` (`id`)
+        ON DELETE CASCADE ON UPDATE NO ACTION,
+    CONSTRAINT `schema_id_idx` FOREIGN KEY (`schema_id`)
+        REFERENCES `schemas` (`id`)
+        ON DELETE NO ACTION ON UPDATE NO ACTION,
+    CONSTRAINT `subjects__constraint_key` UNIQUE (`subject`, `version`, `project_id`)
+)  ENGINE=NDBCLUSTER DEFAULT CHARSET=LATIN1 COLLATE = LATIN1_GENERAL_CS;
```

Comment:
@kai-chi just for consistency, let's keep the values lower cased as in the other tables ` ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;`